packages/snjs/test/lib/model_tests/appmodels.test.js

```diff
@@ -0,0 +1,441 @@
+import { CreateMaxPayloadFromAnyObject, PayloadSource, CopyPayload, ComponentArea } from '@Lib/index';
+import { SNItem, CreateItemFromPayload, ContentType } from '@Lib/models';
+import { EncryptionIntent } from '@Lib/protocol';
+import * as Factory from '../../factory';
+
+describe('app models', () => {
```

Comment:
All applications in this file need to be deinited.

Reply:
Also, can we preserve the original directory for some of these test files so that Git is not treating them as new files but rather changes to existing files?

Reply:
Mmm... I'm not sure why GitHub is not showing a diff for this file. I actually moved this test file [in this commit](https://github.com/standardnotes/snjs/pull/357/commits/03e22a4b2d05a6b27a3aa259c85dd1131f24081c#diff-7cb90e8d955a4fe1dc4b40a7f22326293a90672afb1127abcabef793d73d166f).